internal/pkg/cli/storage_init.go

```diff
@@ -109,7 +109,6 @@ type initStorageOpts struct {
 	ws    wsAddonManager
 	store store
 
-	app *config.Application
```

Comment:
Hmm, I thought this was getting used. These are used elsewhere as a cached value (in `svc deploy` it's `o.targetApp`) but I guess since storage doesn't actually need to validate that the app exists, just that there are local services, we never used it.